server/chat-plugins/scavengers.ts

```diff
@@ -895,11 +898,14 @@ export class ScavengerHunt extends Rooms.RoomGame {
 
 			// notify staff
 			const staffMsg = `(${player.name} has been caught attempting a hunt with ${uniqueConnections} connections on the account. The user has also been given 1 infraction point on the player leaderboard.)`;
-			const logMsg = `([${player.id}] has been caught attempting a hunt with ${uniqueConnections} connections on the account. The user has also been given 1 infraction point on the player leaderboard.)`;
 
 			this.room.sendMods(staffMsg);
 			this.room.roomlog(staffMsg);
-			this.room.modlog(logMsg);
+			this.room.modlog({
+				action: 'SCAVCHEATER',
```

Comment:
why is this `SCAVCHEATER` but the others have a space like `SCAV BLITZ`? are there any guidelines on what these should be called?

Reply:
there really isn't - SCAV CHEATER works fine too